packages/snap-preact-components/src/components/Atoms/NoResults/NoResults.tsx

```diff
@@ -0,0 +1,132 @@
+/** @jsx jsx */
+import { Fragment, h } from 'preact';
+
+import { observer } from 'mobx-react-lite';
+import { jsx, css } from '@emotion/react';
+import classnames from 'classnames';
+
+import { Theme, useTheme } from '../../../providers';
+import { cloneWithProps, mergeProps } from '../../../utilities';
+import { ComponentProps, StylingCSS } from '../../../types';
+import { handleize } from '@searchspring/snap-toolbox/dist/cjs/filters/handleize';
+import { SearchController } from '@searchspring/snap-controller';
+
+const CSS = {
+	noresults: () => css({}),
+};
+
+export const NoResults = observer((properties: NoResultsProps): JSX.Element => {
+	const globalTheme: Theme = useTheme();
+
+	const defaultProps: Partial<NoResultsProps> = {
+		dymText: `Did you mean <a href=${properties.controller?.store?.search?.didYouMean?.url.href}>${properties.controller?.store?.search?.didYouMean?.string}</a>?`,
```

Comment:
Seems like this should maybe be a function? Otherwise how would the text be set properly?

But at the same time - does this really belong here? Or should it really be in the `SearchHeader`?